mobile/apps/artwork/components/bid/templates/bid_button.jade

```diff
@@ -18,13 +18,8 @@ form.artwork-auction-bid-form.js-artwork-auction-bid-form(
           type='button'
           disabled=true
         ) Registration Closed
-      when 'registration-required'
```

Comment:
It seems like we want to show a different button here depending on whether or not you are logged in. Can you clarify what the expectations should be there?